lighthouse-core/report/html/report-styles.css

```diff
@@ -842,6 +842,7 @@ summary.lh-passed-audits-summary {
 
 .lh-table-column--text,
 .lh-table-column--bytes,
+.lh-table-column--timespanMs,
```

Comment:
😢 I guess a _.toKebabCase is a bit much huh?

Reply:
> I guess a _.toKebabCase is a bit much huh?

ha, that's a good point, but it'd also be pretty ugly. Is the tear because you're accepting living with it or do you really want this?

Reply:
💦 because I'm living with it :)